model/database/registry.yaml

```diff
@@ -541,9 +542,50 @@ groups:
               value: "ConsistentPrefix"
               stability: experimental
         stability: experimental
-        brief: Account or request [consistency level](https://learn.microsoft.com/azure/cosmos-db/consistency-levels).
+        brief: >
+          Effective Request [consistency level](https://learn.microsoft.com/azure/cosmos-db/consistency-levels).
+          This represents the first applicable consistency level among those specified by the request or the client.
         examples: ["Eventual", "ConsistentPrefix", "BoundedStaleness", "Strong", "Session"]
 
+  - id: registry.db.cosmosdb.network
+    type: attribute_group
+    display_name: Azure Cosmos DB Network level Attributes
+    stability: experimental
+    brief: >
+      This group defines attributes for Azure Cosmos DB Network call.
+    attributes:
+      - id: db.cosmosdb.network.host
+        type: string
+        stability: experimental
+        brief: Specifies the host address from the Gateway or Backend Service endpoint.
+        examples: ["rntbd://cdb-east-us.azure.document.com"]
+      - id: db.cosmosdb.network.port
+        type: int
+        stability: experimental
+        brief: The port used for the Gateway or Backend Service endpoint.
+        examples: 443
+      - id: db.cosmosdb.network.response.status_code
+        type: int
+        stability: experimental
+        brief: The HTTP status code returned from the network request.
+        examples: 200
+      - id: db.cosmosdb.network.response.sub_status_code
+        type: int
+        stability: experimental
+        brief: The Azure Cosmos DB sub-status code for a network request made during an operation.
+        examples: 1002
+      - id: db.cosmosdb.network.region
+        type: string
+        stability: experimental
+        brief: Specifies the Azure Cosmos DB target region for the network request.
+        examples: "WEST US 2"
+      - id: db.cosmosdb.network.resource_id
```

Comment:
resource_id could be confused with Azure resource id, could we find some other name that would be descriptive?

Also `Azure Cosmos DB Partition Key Range ID for a Gateway call, or Partition ID/Replica ID for a Direct call` does not sound like something network-specific - why put it into `db.cosmosdb.network` namespace?

Reply:
It is network specific information, because SDK always makes a network call to a particular partition id or pkrangeid. what about `cosmos_routing_id`

cc @FabianMeiswinkel @kirankumarkolli

Reply:
I like cosmos_routing_id

Reply:
making it `routing_id`, cosmos is already there in the prefix.